libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -172,13 +177,31 @@ class MapboxRouteLineApi(
      * @return a state which contains the side effects to be applied to the map displaying the
      * newly designated route line.
      */
-    fun updateToPrimaryRoute(route: DirectionsRoute): Expected<RouteSetValue, RouteLineError> {
-        val newRoutes = directionsRoutes.filter { it != route }.toMutableList().also {
-            it.add(0, route)
+    suspend fun updateToPrimaryRoute(route: DirectionsRoute):
```

Comment:
Could you move the `suspend` functions to a file in the internal package (refs https://github.com/mapbox/mapbox-navigation-android/issues/4126)?

Reply:
A customer's request for a suspend function was the motivation for this work. Moving the suspend function to an internal package implies it's not accessible to external developers. I"ll move it to an external file but not in an internal package for now and let's discuss further.